samples/spring-framework-petclinic-12.2.1.4/README.md

```diff
@@ -40,13 +40,13 @@ Use any of the following deployment tools for deploying the PetClinic applicatio
 
 #### WebLogic Remote Console
 
-You can use [WebLogic Remote Console](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-remote-console/administer/set-console.html) to manage the deployment process of applications to WebLogic Server. For more information, see [Deploying Applications](https://docs-uat.us.oracle.com/en/middleware/fusion-middleware/weblogic-remote-console/administer/deploying-applications.html) in the _Oracle WebLogic Remote Console Online Help_.
+You can use the [WebLogic Remote Console](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-remote-console/administer/set-console.html) to manage the deployment process of applications to WebLogic Server. For more information, see [Deploying Applications](https://docs-uat.us.oracle.com/en/middleware/fusion-middleware/weblogic-remote-console/administer/deploying-applications.html) in the _Oracle WebLogic Remote Console Online Help_.
 
 For general information about the application deployment process, see [Understanding WebLogic Server Deployment](https://docs.oracle.com/pls/topic/lookup?ctx=en/middleware/fusion-middleware/weblogic-remote-console/administer&id=DEPGD-GUID-F6E8BF0B-FBCF-44D2-A33F-13C4EF2E0031) in _Deploying Applications to Oracle WebLogic Server_.
 
 Installing an application makes its physical file or directory known to WebLogic Server.
 
-This procedure applies to all of the deployment units listed in Supported Deployment Units.
+This procedure applies to all of the deployment units listed in [Supported Deployment Units](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/14.1.2/depgd/understanding.html#GUID-DC6C0B59-7560-4A6F-964B-201480072A3D).
```

Comment:
This sample was created for WLS 12.2.1.4, may need both documentation reference.

Reply:
My Mistake. I'll change it to the 12.2.1.4.0 docs.